apps/protocol-api/sql/add_membership_status.sql

```diff
@@ -1,5 +1,6 @@
 INSERT INTO "GuildMembershipStatus" (name)
-VALUES ('Member'),
+VALUES ('Admin'),
```

Comment:
Based on this [doc](https://hackmd.io/SDOyv9B4RCaNiLa1MpWuaQ), I added the Admin role.
If this is not the case, I can undo this commit 606b83f978048624d2f3948446fb283cac4aeedd.